Doc/library/ast.rst

```diff
@@ -145,11 +145,12 @@ and classes for traversing abstract syntax trees:
    modified to correspond to :pep:`484` "signature type comments",
    e.g. ``(str, int) -> List[str]``.
 
-   Also, setting ``feature_version`` to the minor version of an
-   earlier Python 3 version will attempt to parse using that version's
-   grammar.  For example, setting ``feature_version=4`` will allow
-   the use of ``async`` and ``await`` as variable names.  The lowest
-   supported value is 4; the highest is ``sys.version_info[1]``.
+   Also, setting ``feature_version`` to a tuple ``(major, minor)``
+   will attempt to parse using that Python version's grammar.
+   Currently ``major`` must equal to ``3``.  For example, setting
+   ``feature_version=(3, 4)`` will allow the use of ``async`` and
+   ``await`` as variable names.  The lowest supported version is
+   ``(3, 4)``; the highest is ``sys.version_info[0:2]``.
```

Comment:
Note that there's a subtle difference between "must equal to 3" (the call fails if it isn't) and "supported" -- versions outside the supported range are treated as the lowest or highest supported version, respectively.